wordpress-mobile/WordPress-iOS
# People Management: Spinner

Fixes #5276

To test:
1. Fresh install WPiOS
2. Log into a Dotcom account
3. Open `My Sites` > `Site` > `People`

As a result, a spinner should appear onscreen. On completion, it should get dismissed.
Plus: whenever the `People` screen gets displayed, the list of users will be refreshed. The spinner will only be visible when the list is empty.

Needs review: @diegoreymendez 
Thanks in advance Diego!
